js/recipe/metadata.js

```diff
@@ -0,0 +1,31 @@
+import { htmlElement } from "../htmlRenderer.js"
+import { els } from "../htmlRenderer.js"
+import { render } from "../htmlRenderer.js"
+
+export function metadata(recipe) {
+  // Recipe Data
+  const tags = recipe.tags;
+  const title = recipe.name;
+  const publishedAt = new Date(recipe.datePublished).toDateString();
+  const serves = recipe.recipeYield;
+
+  const recipeParentElement = document.querySelector("article.post.single");
+
+  // TITLE
+  render(recipeParentElement, htmlElement("h1")({className: "post-title", innerHTML: title}));
+
+  // METADATA
+  const metaParentElement = render(recipeParentElement, els.div({className: "post-meta clear"}));
```

Comment:
I think this is also relying on the fact that `render`-ing to the same target element twice puts the elements next to each other. I'd suggest we change `render` to clear the target element first, then think about how we may declare that `h1` and `div` sit side by side - calling `render` once.
Ideally we could only call `render` a handful of times in all of this JS (similar to Banking React code), but we can gradually move towards that.

Reply:
Yes, I can use the same approach that I used in introduction.js to render it once, with all the children passed in in one go

Reply:
I've managed to get most of the metadata elements rendered at once by creating an array of `metaChildren` elements and passing them in when I render the parent div. The title element is not included in this however because it's not a child node of the parent metadata element.

I'm wondering how to do this and I think we could modify render to take a `siblings` argument, and then use appendChild to add those to the target at the end of the render method?